system-tests/__snapshots__/system_node_spec.js

```diff
@@ -5,11 +5,10 @@ exports['e2e system node uses system node when launching plugins file 1'] = `
   (Run Starting)
 
   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
-  │ Cypress:        1.2.3                                                                          │
-  │ Browser:        FooBrowser 88                                                                  │
-  │ Node Version:   vX (/foo/bar/node)                                                             │
```

Comment:
It still might be useful to log this message for anyone that might spawn this programmatically or provide an override version via the CLI.

Reply:
The presence of the node version message is based on whether `resolvedNodePath` is undefined or not. In this case since the test is not initiated via the cypress cli, there isn't a userNodePath or version defined so the test falls back to the bundled node version. So this feature isn't removed it's just not being triggered in this test.

https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/modes/run.js#L151

Reply:
I figured out a better way to fix this test so the snapshot should be the same now.